packages/blade/src/tokens/global/typography.ts

```diff
@@ -0,0 +1,101 @@
+import type { FontFamily } from './fontFamily';
+import fontFamily from './fontFamily';
+
+type FontWeight = {
+  regular: 400;
+  bold: 700;
+};
+
+type FontSize = {
+  10: number;
+  25: number;
+  50: number;
+  75: number;
+  100: number;
+  200: number;
+  300: number;
+  400: number;
+  500: number;
+  600: number;
+  700: number;
+  800: number;
+  900: number;
+  1000: number;
+};
+
+const fontWeight: FontWeight = {
+  regular: 400,
+  bold: 700,
+};
+
+export type Typography = Record<
```

Comment:
Haven't we incorporated the main scale yet? I mean - [Desktop/Mobile Scale](https://www.figma.com/file/LV9kjgzhkfsktvbRgTuGJ3/Blade-DSL?node-id=5%3A0)

Reply:
We have, you're looking at a TypeScript type 😄 
See [here](https://github.com/razorpay/blade/pull/276/files#diff-efc5afda90b97389459637e4fbe63fcfb21a7590bd46f798e5bcc9d9911a334fR43-R48)